app/templates/components/footer-main.hbs

```diff
@@ -72,5 +72,12 @@
         </div>
       {{/ui-dropdown}}
     </div>
+    <div class="sixteen wide column">
```

Comment:
@sumedh123 Please remove the dummy links.
![screen shot 2017-08-25 at 3 37 51 pm](https://user-images.githubusercontent.com/12807846/29709850-a10a80d8-89ab-11e7-9480-8ea149f93bd8.png)

Reply:
@geekyd Done :+1: 
![pages-footer](https://user-images.githubusercontent.com/17925669/29710294-4db86c5e-89ad-11e7-9c2e-c07f2954d225.png)